.gitignore

```diff
@@ -426,3 +426,7 @@ tests/integration/ui/cypress/screenshots/
 
 #JetBrains IDE
 .idea/
+package.json
+package-lock.json
+package.json
+package.json
```

Comment:
Please remove lines 429 through 432. These should not be in the .gitignore file.